frontend/src/metabase/visualizations/components/settings/ChartSettingTableColumns/TableColumnPanel/utils.ts

```diff
@@ -14,6 +15,7 @@ import type { ColumnItem } from "./types";
 export function getColumnItems(
   columns: DatasetColumn[],
   columnSettings: TableColumnOrderSetting[],
+  tc?: ContentTranslationFunction,
```

Comment:
if you make this default to `_.identity` then you don't have to check for `tc` exists